embeddings/model/lightning_model.py

```diff
@@ -13,17 +15,21 @@ def __init__(
         self,
         trainer: pl.Trainer,
         task: HuggingFaceLightningTask,
-        predict_subset: Literal["dev", "test"] = "test",
+        predict_subset: LightingDataModuleSubset,
```

Comment:
predict dataloader is not defined in datamodule although we allow to use it for predicting; it also does not make sense to predict on train dataloader

Reply:
It's universal class, predict data loader should be allowed although

Reply:
Currently we don't support cases whether we want to  skip prediction step, so passing train data loader can be used for such case.

Reply:
ok